source/js/main.js

```diff
@@ -0,0 +1,107 @@
+import { storage } from "./storage.js";
+
+window.addEventListener("DOMContentLoaded",init);
+
+const gliderConfig = {
+  focusAt: 'center', //this line seems to being nothing. i wanted it to maybe like, enable the non-translucence. or something like that
+  type: 'carousel',
+  perView: 3,
```

Comment:
Codacy has a fix for the issue: [Expected object keys to be in ascending order. 'perView' should be before 'type'.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8437408)
```suggestion
  //this line seems to being nothing. i wanted it to maybe like, enable the non-translucence. or something like that
```